indigo_app/static/lib/indigo-web/scss/_coverpage.scss

```diff
@@ -55,6 +55,7 @@
       font-style: italic;
     }
 
+    .commencement-note,
```

Comment:
this will need to happen in indigo-web too, otherwise the LIIs etc. won't pick it up.